docs/pages/develop/development-builds/create-a-build.mdx

```diff
@@ -14,17 +14,47 @@ import { BookOpen02Icon } from '@expo/styleguide-icons';
 
 Development builds can be created with [EAS Build](/build/introduction/) or [locally](/guides/local-app-development/) on your computer if you have Android Studio and Xcode installed.
 
-<Video url="https://www.youtube.com/embed/LUFHXsBcW6w" />
+In this guide, you'll find information on creating a development build and installing it on an emulator/simulator or a physical device to continue developing your app.
```

Comment:
```suggestion
In this guide, you'll find information on creating a development build with EAS and installing it on an emulator/simulator or a physical device to continue developing your app.
```

do we have somewhere we can link to about creating dev builds locally?

Reply:
We don't explicitly have info that calls out or explains dev builds locally. I think we can do that within this guide. I need to confirm the process of creating a dev build locally will involve:
- installing `expo-dev-client`
- then running prebuild commands to compile the debug build?

Reply:
No soy programador están robando información de mi dispositivo se cierran las ventanas de Android

Reply:
@brentvatne, I've updated Local app development guide to include the following section, which tells the developers to install the library and use `npx expo run` compilation commands. 
<img width="865" alt="CleanShot 2023-10-23 at 22 43 24@2x" src="https://github.com/expo/expo/assets/10234615/5916f9c5-b1f9-4bd8-a649-1ed2d59bdd4e">


I did not feel like adding this section in the Create Development Build guide since that is the recommended path. However, I've expanded the introduction of this doc and provided links for both local builds with `expo-dev-client` library and locally with EAS.

 
<img width="798" alt="CleanShot 2023-10-23 at 22 42 48@2x" src="https://github.com/expo/expo/assets/10234615/0e5da493-82f1-46ea-8cc7-4ccef8947f5f">

Let me know what you think.